### PR TITLE
Enable ruff check in CI; fix & ignore current issues found by ruff

### PR DESCRIPTION
**Pull request by KonstantAnxiety**

### CODE REVIEW

File: lib/dl_api_lib/dl_api_lib/app/control_api/resources/info.py [hunk 1]
```diff
@@ -130,12 +130,12 @@ def get(self, conn_type: str, form_mode: str) -> dict:
         try:
             ct = ConnectionType(conn_type)
         except ValueError:
-            raise UnsupportedForEntityType(f"Unknown connector type: {conn_type}")
+            raise UnsupportedForEntityType(f"Unknown connector type: {conn_type}") from None
```
Comment (ovsds) [inline lib/dl_api_lib/dl_api_lib/app/control_api/resources/info.py:133]: let's myte this one (not line but the whole check)

File: lib/dl_pivot/dl_pivot/native/sorter.py [hunk 2]
```diff
@@ -100,7 +100,7 @@ def _sort_by_measure(self, axis: SortAxis, sorting_piid: int, settings: PivotMea
         sorting_dim_values: Optional[tuple[DataCellVector, ...]] = None
         other_axis = self._complementary_axis(axis)
 
-        for idx, header in enumerate(self._pivot_dframe.iter_axis_headers(axis)):
+        for _idx, header in enumerate(self._pivot_dframe.iter_axis_headers(axis)):
```
Comment (ovsds) [inline lib/dl_pivot/dl_pivot/native/sorter.py:103]: mb just drop enumerate if _idx is left unused

File: lib/dl_api_lib/dl_api_lib/app/control_api/resources/info.py (hunk 1 above)
Comment (KonstantAnxiety) [reply lib/dl_api_lib/dl_api_lib/app/control_api/resources/info.py:133]: Sure
Can we keep what I've added?

Comment (ovsds) [reply lib/dl_api_lib/dl_api_lib/app/control_api/resources/info.py:133]: why not, just let's mute it for future generations to come

**Review by ovsds — APPROVED** [review]

**Pull request merged**